doc/release-notes/Makefile.am

```diff
@@ -12,12 +12,11 @@ DOC= release-$(SQUID_RELEASE)
 
 SUFFIXES= .sgml.in .sgml
 
-.sgml.in.sgml:
-	sed \
-        -e "s%[@]SQUID_VERSION[@]%$(VERSION)%g" \
-        -e "s%[@]SQUID_RELEASE[@]%$(SQUID_RELEASE)%g" \
-        -e "s%[@]SQUID_RELEASE_OLD[@]%$$(( $(SQUID_RELEASE) - 1 ))%g" \
-        < $< >$@
+.sgml.in.sgml: mk-release-notes.awk
+	SQUID_VERSION="$(VERSION)" \
+	PACKAGE_VERSION="$(PACKAGE_VERSION)" \
+	SQUID_RELEASE="$(SQUID_RELEASE)" \
+	$(AWK) -f $(srcdir)/mk-release-notes.awk $< >$@ || ($(RM) -f $@ && exit 1)
 	test `grep -c "@SQUID" $@` -eq 0
```

Comment:
Isn't this the same as 
`$(FGREP) "@SQUID" $@ >/dev/null`
?

Reply:
I'm not sure we need that line at all really. Especially now that the `|| ...` handler exists. If we agree to remove or change it I am fine with that. Otherwise I will leave out of scope.